packages/agreement-process/test/upgradeAgreement.test.ts

```diff
@@ -554,7 +554,7 @@ describe("upgrade Agreement", () => {
     }
   });
 
-  it("should succeed with valid Verified, Certified, and Declared attributes when consumer and producer are different and requester is a delegate", async () => {
+  it("should succeed with valid Verified, Certified, and Declared attributes when consumer and producer are different, requester is consumer, an active producer delegation exists and is taken into account for the PDF contract generation", async () => {
```

Comment:
Unrelated, renaming this test because the upgrade cannot be called by a producer or producer delegate - the test tests that the producer delegate is taken into account when generating the contract